network/src/main/protobuf/driver_service.proto

```diff
@@ -128,8 +128,9 @@ message BSessionHandle {
 // Client-side reference to a task running
 // asynchronously on the server.
 message BOperationHandle {
-    BHandleIdentifier operation_id = 1;
-    BOperationType operation_type = 2;
+    BSessionHandle session_handle = 1;
```

Comment:
Here should use session_handle to get cached sessions, but may be other good ways?